### ISSUE TITLE
server downtime due to hardware failure

### ISSUE THREAD

**Issue by Meghanaa020299** [Meghanaa020299/assignment3#9]
System outages or downtime caused by hardware failures, resulting in service disruptions, data inaccessibility, and potential loss of productivity.
